expressions/aggregation/AggregationHandle.hpp

```diff
@@ -265,7 +265,8 @@ class AggregationHandle {
    **/
   virtual ColumnVector* finalizeHashTable(
       const AggregationStateHashTableBase &hash_table,
-      std::vector<std::vector<TypedValue>> *group_by_keys) const = 0;
+      std::vector<std::vector<TypedValue>> *group_by_keys,
+      int index) const = 0;
```

Comment:
Can you please update the doxygen for the ``index`` parameter? Also, if the index is always positive, can you change its type from int to ``std::size_t``?